js/adapt-contrib-xapi.js

```diff
@@ -626,9 +626,12 @@ define([
           break;
         }
         case 'matching': {
-          response = response.replace(/#/g, "[,]");
-          response = response.replace(/\./g, "[.]");
-
+          response = response
```

Comment:
might be useful to give an example of what the conversion is doing so that it's not necessary to parse the code to figure it out?